### ISSUE TITLE
On phone: improve order of sections

### ISSUE THREAD

**Issue by pitag-ha** [ocaml/v2.ocaml.org#1306]
This would be a **good first issue** for anyone looking to get started improving and contributing to OCaml.org.

On the phone, the sections of the page are positioned in a top-down manner. Currently, the last three sections are *News*, *A taste of OCaml* and *Packages* (in that order). I suggest to swap the order of *News* and *A taste of OCaml*. 

The reason is that the expectation of entering a website about OCaml is to get some idea about the language and *A taste of OCaml* offers that. The website's *News* section tends to be towards the end of a website.

**Comment by patricoferris** [comment]
Improving this user experience would be great, thanks for raising the issue :)) 

@shlokalakka would you like to work on this as part of #1245? Please let us know either way below.

**Comment by jyotibalodhi** [comment]
Hello @patricoferris I am an outreachy applicant this year and would like to work on the issue. If @pitag-ha is not already, then would be happy to proceed forward.

**Comment by pitag-ha** [comment]
@jyotibalodhi, could you please wait for @shlokalakka to answer if they want to work on this issue? :) If they say no, you can certainly work on it, but we should give them some time to answer first.

We're adding more issues at the moment so that everyone who wants to contribute can do so. In general, if there's no open `good-first-issue`, you can let us know [here](https://github.com/ocaml/ocaml.org/issues/1245).

**Comment by jyotibalodhi** [comment]
> @jyotibalodhi, could you please wait for @shlokalakka to answer if they want to work on this issue? :) If they say no, you can certainly work on it, but we should give them some time to answer first.
> 
> We're adding more issues at the moment so that everyone who wants to contribute can do so. In general, if there's no open `good-first-issue`, you can let us know [here](https://github.com/ocaml/ocaml.org/issues/1245).

Hello @pitag-ha , Certainly. I probably missed the mention! :)

**Comment by shlokalakka** [comment]
hey @kaniaganda
You can work on this Issue as I'm not attempting it.

**Comment by kaniaganda** [comment]
@shlokalakka Yes! I would be happy to work on this issue.
@pitag-ha and @patricoferris Is it fine if I work on this issue, or is @jyotibalodhi working on this issue?

**Comment by jyotibalodhi** [comment]
> @shlokalakka Yes! I would be happy to work on this issue.
> @pitag-ha and @patricoferris Is it fine if I work on this issue, or is @jyotibalodhi working on this issue?

Hi @kaniaganda , as @shlokalakka is not working on the issue, I think I would like to work on it. I would however definitely reach out and mention if I have any problems and need help :)

**Comment by kaniaganda** [comment]
@jyotibalodhi No problem! Thanks for letting me know that you are working on the issue. I would be happy to contribute if help is still needed in the future.

**Comment by jyotibalodhi** [comment]
Hi @pitag-ha @patricoferris, I have completed the local built of OCaml project. But while exploring the codebase, I am not able to find the correct location to make changes to for solving this particular issue. I have gone through the contributing guidelines, which state the script and site folders of the project contain most of these files but unable to find them. I would be really glad if you could help me out with that, as I see many markdown and other files. Also, as I am planning to work on other issues as well, could you let me know if there's a pattern to the files being stored in some location so that it will be easy for me to understand the codebase better. Thanks in advance! :)

**Comment by pitag-ha** [comment]
Hi @jyotibalodhi , a typical approach to find a section would be for example to search for some plain text in it. For example, I would guess that to find out where the `Taste of OCaml` section is, you could search for "OCaml is a lot more powerful than this simple example shows." or some part of it. You could do the search for example on the terminal with `grep` or in an editor like VSCode with Ctrl + Shift + F (not sure about other editors). Or did you mean something different? :)

**Comment by jyotibalodhi** [comment]
> Hi @jyotibalodhi , a typical approach to find a section would be for example to search for some plain text in it. For example, I would guess that to find out where the `Taste of OCaml` section is, you could search for "OCaml is a lot more powerful than this simple example shows." or some part of it. You could do the search for example on the terminal with `grep` or in an editor like VSCode with Ctrl + Shift + F (not sure about other editors). Or did you mean something different? :)

Yes, I was referring to this problem only. Thanks for the help @pitag-ha

**Comment by jyotibalodhi** [comment]
Hello @pitag-ha @patricoferris I am facing a problem while working on the issue, and I am hoping you could help me with that.  On my local machine, when I built and open the site with live server it is showing me some very different errors in the layout. Although my working directory is clean, and there are no changes at all. As I tried viewing it with different branches, and even after removing all the changes that I made for the issue. So I was wondering whether it is something with my local machine or I am doing something wrong somewhere. I am attaching two screenshots for a better understanding of my problem. Thanks in advance! 
![err1](https://user-images.githubusercontent.com/63946672/113326171-f1d95000-9336-11eb-8ea3-955336c3ca8d.png)
![err2](https://user-images.githubusercontent.com/63946672/113326180-f56cd700-9336-11eb-83ea-be48e7fb0016.png)

**Comment by patricoferris** [comment]
Hi @jyotibalodhi, this problem is quite hard to diagnose but hopefully the following can help: 

 - To check if it might be something wrong with the build or a consequence of a change you made, the best thing to do, is to stash your changes and then run `git checkout master`, remove all of the built files with `make clean` then re-run `make` and see if the changes persist.
 - If you are still struggling or unsure if you might have caused it, you can always push your code to a branch of your fork and ask us to take a look. This makes it much easier to understand the changes than just images or even code snippets. 
 
Hopefully we can get this resolved soon :))

**Comment by jyotibalodhi** [comment]
> * To check if it might be something wrong with the build or a consequence of a change you made, the best thing to do, is to stash your changes and then run `git checkout master`, remove all of the built files with `make clean` then re-run `make` and see if the changes persist.

Okay, I will try doing that, and update if still the problem continues. Thank you @patricoferris :)

**Comment by jyotibalodhi** [comment]
> * To check if it might be something wrong with the build or a consequence of a change you made, the best thing to do, is to stash your changes and then run `git checkout master`, remove all of the built files with `make clean` then re-run `make` and see if the changes persist.

Hi @patricoferris I tried this solution but apparently, I am getting a new error while making the file again. What do you think went wrong.

![1](https://user-images.githubusercontent.com/63946672/113416432-f35e5300-93de-11eb-88f6-3f9e0bc88087.png)

**Comment by patricoferris** [comment]
Ah, you will need to rerun `make deps`. We have since updated `cohttp` to version `4.0.0` (you might need to run `opam update` too if it can't find it). After it has installed version `4.0.0` you should be good to go :))

**Comment by jyotibalodhi** [comment]
Hello @pitag-ha @patricoferris For working on this issue I have tried using flexbox order property and a few other methods but nothing seems to work as the two sections that need to be swapped are nested in different divs. Can you plz help me out with what other solutions I can try? Also, I was wondering if I can position them hard-coded as I saw a few other elements being done like that in the bootstrap file. What do you think about this? Much appreciated!! Thanks :)

**Comment by patricoferris** [comment]
Hi @jyotibalodhi, 

So I've thought a little about this. The only way to make the changes is to either do some complicated JS code to rearrange the content on mobile, do some complicated CSS acrobatics or to change the actual ordering for all devices. I agree with @pitag-ha's idea though.

One suggestion (we'll have to see what the core maintainers think) is to change `A Taste of OCaml` and the `News` section on **all** devices? The column widths might have to big changed a bit but even desktop I think it would be nice to see OCaml code the minute you land on the home page? @jyotibalodhi, @pitag-ha what do you think?

**Comment by pitag-ha** [comment]
I agree. I think `A Taste of OCaml` is an important section, so to me it sounds good moving it up even on other devices than phones.

**Comment by jyotibalodhi** [comment]
@patricoferris  I do think it's a good idea. And yes the column width needs to be changed if that happens but that is still a easy task. And as you mentioned that we don't want JS to be included for just swapping then I think this is probably the safest option to go along with.

**Comment by jyotibalodhi** [comment]
Hello @patricoferris @pitag-ha , should I proceed with what we discussed in earlier comments, or I shall wait for the maintainer's response on this? 
Kindly let me know, thanks ! :)
